src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -84,6 +84,18 @@
 				id="dangerzone"
 				:name="t('spreed', 'Danger zone')">
 				<LockingSettings v-if="canFullModerate && !isNoteToSelf" :token="token" />
+				<h4 class="app-settings-section__subtitle">
+					{{ t('spreed', 'Archive conversation') }}
+				</h4>
+				<NcCheckboxRadioSwitch v-if="supportsArchive"
+					type="switch"
+					:checked="isArchived"
+					@update:checked="toggleArchiveConversation">
+					{{ t('spreed', 'Archive conversation') }}
+				</NcCheckboxRadioSwitch>
+				<p class="app-settings-section__hint">
+					{{ t('spreed', 'Archived conversations are hidden from the conversation list by default. They will still be shown, if you have been mentioned directly, when you search for the conversation name or filter your conversation list for archived conversations.') }}
```

Comment:
```suggestion
					{{ t('spreed', 'Archived conversations are hidden from the conversation list by default. They will still be shown when you search for the conversation name or open archived conversations list.') }}
```

Because mentioning depends on user notification settings

Reply:
Dropping line for now to not create extra work for translators. Will continue in follow-up